src/Tickets/Commerce/Ticket.php

```diff
@@ -843,30 +872,29 @@ public function decrease_ticket_sales_by( $ticket_id, $quantity = 1, $shared_cap
 	}
 
 	/**
-	 * Gets the product price value.
-	 *
-	 * @todo  TribeCommerceLegacy: This should not be used, the model should be used.
+	 * Gets the product price value object
 	 *
-	 * @since 5.1.9
+	 * @since   5.1.9
+	 * @since   TBD method signature changed to return an instance of Value instead of a string.
 	 *
 	 * @param int|\WP_Post $product
 	 *
-	 * @return string
+	 * @return Commerce\Utils\Value;
+	 * @version TBD
+	 *
 	 */
 	public function get_price_value( $product ) {
-		$product = get_post( $product );
+		$ticket = Models\Ticket_Model::from_post( $product );
```

Comment:
Use `tec_tc_get_ticket`

Reply:
@bordoni `tec_tc_get_ticket`  retuns the `WP_Post` object of the ticket, right? We need the model instance here, not the post object, so we can create a new Value object from it.